core/libs/install-libs.sh

```diff
@@ -7,23 +7,30 @@ BUILD_PACKAGES=(
     g++
 )
 
-apt update
-apt install -y --no-install-recommends ${BUILD_PACKAGES[*]}
+echo "Target architecture: $TARGETARCH"
+echo "Target variant: $TARGETVARIANT"
+# Install build packages if not on armv7 or amd64, which are the only platforms that have pre-built wheels
+if ! { [ "$TARGETARCH" == "arm" ] && [ "$TARGETVARIANT" == "v7" ]; } && [ "$TARGETARCH" != "amd64" ]; then
+    apt update
+    apt install -y --no-install-recommends ${BUILD_PACKAGES[*]}
```

Comment:
Should we remove this at all ? Everything else should have pre-built binaries, no ? piwhell will mostly ensure that.

Reply:
oddly enough, there were binaries missing for x86_64. IIRC it was aiohttp